modules/vpc/main.tf

```diff
@@ -0,0 +1,3 @@
+resource "aws_vpc" "devops-vpc" {
+  cidr_block = var.cidr
+}
```

Comment:
Per https://github.com/nvminhtue/tyson_devops_ic/pull/18#discussion_r1360557922 are we keeping this just temporary as the last comment? because we will use module from AWS not resource, and the name be different as per namespace comment.